tests/utils.go

```diff
@@ -4576,3 +4576,18 @@ func GetKubeVirtConfigMap() (*k8sv1.ConfigMap, error) {
 
 	return cfgMap, err
 }
+
+func ClearKubeVirtConfigMap(key string) error {
+	virtClient, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
+
+	cfgMap, err := GetKubeVirtConfigMap()
+	if err == nil {
+		if _, ok := cfgMap.Data[key]; ok {
+			data := fmt.Sprintf(`[{ "op": "remove", "path": "/data/%s"}]`, key)
```

Comment:
This is likely good enough, but because we're messing about with KubeVirt's ConfigMap here, perhaps we should be slightly paranoid:
```
		if val, ok := cfgMap.Data[key]; ok {
                        testJson := fmt.Sprintf(`[{ "op": "test", "path": "/data/%s", "value": %s}]`, key, val)
			removeJson := fmt.Sprintf(`[{ "op": "remove", "path": "/data/%s"}]`, key)
                        data := fmt.Sprintf("[%s, %s]", testJson, removeJson)
			_, err = virtClient.CoreV1().ConfigMaps(KubeVirtInstallNamespace).Patch(virtconfig.ConfigMapName, types.JSONPatchType, []byte(data))
```

Note: I just scribbled that snippet in this comment box. it's untested. Basically the gist here is that we should send a patch that compares the configMap state to what we expect it to be before removing the key. I can't think of a scenario where this could actually happen to trip us, but coding it this way will trigger an error if the cluster's in a state we didn't expect.

Reply:
@stu-gott I did exactly what you mentioned but the JSON test failed although the key was there. I could not understand how to do JSON test for existence of keys, not values.

Reply:
Looks like JSON test not works of intermediate nodes but only for leaves, if thinking of JSON as of an analogy to a tree.